### ISSUE TITLE
SVM with nonconvex

### ISSUE THREAD

**Issue by beignner** [cvxgrp/dccp#56]
Thanks for your attention.
I am trying to solve an objective function  Support vector machine with a truncated hinge loss.
The objective function as follow:
![image](https://user-images.githubusercontent.com/49185490/86513867-3edbd480-be40-11ea-8cd0-c6a8dc3daf50.png)
![image](https://user-images.githubusercontent.com/49185490/86513883-59ae4900-be40-11ea-9655-cdebb7bf1a9c.png)

H1-Hs is the truncated hinge loss which represented as the green line in the plot.
some literature solved this function by using a d.c. algorithm. 
I coded this question as follows. Since I do not understand the problem and the algorithm deeply, "dccp is False" return. 
I am not sure if the problem can be solved with the DCCP package.
Thanks again for your attention  and your contribution.

beta = cp.Variable((n,1)) # n feature numbers
v = cp.Variable()
h1=cp.pos(1 - cp.multiply(Y,(X @ beta - v)))
hs=cp.pos(-1 - cp.multiply(Y,(X @ beta - v)))
loss = cp.sum(h1-hs)
reg = cp.norm(beta, 1) # beta l1 norm
lambd = cp.Parameter(nonneg=True) # lambda>=0
prob = cp.Problem(cp.Minimize(loss/m + lambd*reg))

prob.is_dcp() # False
dccp.is_dccp(prob) # False

**Comment by SteveDiamond** [comment]
You can do
```
beta = cp.Variable((n,1)) # n feature numbers
v = cp.Variable()
h1=cp.pos(1 - cp.multiply(Y,(X @ beta - v)))
hs=cp.pos(-1 - cp.multiply(Y,(X @ beta - v)))
t = cp.Variable(hs.shape)
loss = cp.sum(h1-t)
reg = cp.norm(beta, 1) # beta l1 norm
lambd = cp.Parameter(nonneg=True) # lambda>=0
prob = cp.Problem(cp.Minimize(loss/m + lambd*reg), [t == hs])
```
